### ISSUE TITLE
Persist Button Doesn't Reflect it's State

### ISSUE THREAD

**Issue by ksloan** [zalmoxisus/redux-devtools-extension#280]
The persist button looks the same for me whether it's active or not.
Is anyone else seeing this?

**Comment by zalmoxisus** [comment]
Yes, that's intended and stated in #276 and #274. We will be able to implement it in `3.0`, with [the new UI](https://github.com/zalmoxisus/remotedev-monitor-components).

**Comment by tnrich** [comment]
@zalmoxisus any news on the 3.0? Seems like it has been a while coming.. Could I just send in a PR patching the current UI to have the toggle state be visible?

Thanks!

**Comment by geon** [comment]
@tnrich : Please do it. It seems @zalmoxisus is just busy with other stuff. Accepting a complete pull request is quick.

**Comment by AndyBarron** [comment]
This is indeed still an issue. :(

**Comment by kamronbatman** [comment]
@tnrich - If you can submit a PR, please do. I will definitely upvote it. Been burned, don't need to be burned more.
